.github/actions/certify-openshift-images/entrypoint.sh

```diff
@@ -9,7 +9,7 @@ DIGESTS=$(docker manifest inspect "${REPOSITORY}:${VERSION}" | jq -r .manifests[
 for DIGEST in $DIGESTS; do
   echo "Checking image $DIGEST"
   # Do the preflight check first
-  preflight check container "${REPOSITORY}:${VERSION}@${DIGEST}" --artifacts "${DIGEST}" --docker-config="${HOME}/.docker/config.json"
+  preflight check container "${REPOSITORY}@${DIGEST}" --artifacts "${DIGEST}" --docker-config="${HOME}/.docker/config.json"
```

Comment:
@helderjs the `${REPOSITORY}` is missing the `quay.io/` prefix (https://github.com/mongodb/mongodb-atlas-kubernetes/actions/runs/4264934609/jobs/7423653842#step:6:463) I think it should be set in the workflow.